src/Build/Construction/Solution/SolutionFile.cs

```diff
@@ -415,7 +415,7 @@ internal static string ParseSolutionFromSolutionFilter(string solutionFilterFile
         {
             try
             {
-                JsonDocument text = JsonDocument.Parse(File.ReadAllText(solutionFilterFile));
+                JsonDocument text = JsonDocument.Parse(File.ReadAllText(solutionFilterFile), new JsonDocumentOptions() { AllowTrailingCommas = true, CommentHandling = JsonCommentHandling.Skip});
```

Comment:
I'd go ahead and create a variable for `JsonDocumentOptions` so that you can comment that it should match the VS behavior.